nemo/collections/nlp/models/lm_model.py

```diff
@@ -47,17 +52,26 @@ def output_types(self) -> Optional[Dict[str, NeuralType]]:
             'nsp_logits': self.nsp_classifier.output_types['logits'],
         }
 
-    def __init__(self, config_file: Optional[str] = None, pretrained_model_name: Optional[str] = 'bert-base-uncased'):
+    def __init__(
+        self,
+        config_file: Optional[str] = None,
+        pretrained_model_name: Optional[str] = 'bert-base-uncased',
+        preprocessing_args: Optional[Dict[str, str]] = None,
+    ):
         """
         Args:
             config_file: model config file
             pretrained_model_name: BERT model name 
+            preprocessing_args: preprocessing parameters for on the fly data preprocessing
         """
         super().__init__()
+        self.pretrained_model_name = pretrained_model_name
+        self.__tokenizer = None
+        if preprocessing_args:
```

Comment:
Why are you using double __ for tokenizer?

Reply:
i followed PTL naming convention, but you are right its confusing since tokenizer is not used by PTL. changed this thx!